### PR TITLE
[all-docker-all-the-time] Use our base Docker image in CI

### PR DESCRIPTION
**Pull request by wlonk**
sea lion

### CODE REVIEW

**Comment by wlonk** [comment]
This uses our base image, but it doesn't use _project's_ image. Maybe it should? Still a WIP.

File: .circleci/config.yml [hunk 1]
```diff
@@ -63,14 +66,18 @@ jobs:
...
+      VIRTUAL_ENV: ./venv
     steps:
       - checkout
       - attach_workspace:
           at: .
       - run:
           name: Test JS
-          command: yarn test:coverage
+          command: |
+            . venv/bin/activate
```
Comment (jgerigmeyer) [inline .circleci/config.yml]: Why does the venv need to be active to run `yarn`?

File: .circleci/config.yml [hunk 2]
```diff
@@ -35,8 +36,8 @@ jobs:
       - persist_to_workspace:
           root: .
           paths:
-            - ./venv
-            - ./node_modules
+            - venv
+            - node_modules
```
Comment (jgerigmeyer) [inline .circleci/config.yml]: Are these necessary changes, or side-effects from lots of small tweaks trying to get CI to pass?

File: .circleci/config.yml [hunk 3]
```diff
@@ -16,9 +18,8 @@ jobs:
           name: Install Dependencies
           command: |
             yarn
-            python3 -m venv venv
+            python3 -m venv venv 
             . venv/bin/activate
-            pip install pipenv
             pipenv install --dev
```
Comment (jgerigmeyer) [inline .circleci/config.yml]: Should we add `--ignore-pipfile` here to match the project Dockerfile?

Comment (wlonk) [reply .circleci/config.yml]: Yeah, I removed it while trying to debug.

File: .circleci/config.yml (hunk 2 above)
Comment (wlonk) [reply .circleci/config.yml]: The latter.

File: .circleci/config.yml (hunk 1 above)
Comment (wlonk) [reply .circleci/config.yml]: It doesn't, here. This was just a side-effect of thrashing.

File: .circleci/config.yml [hunk 4]
```diff
@@ -3,7 +3,9 @@ jobs:
   build:
     working_directory: ~/metashare
     docker:
-      - image: circleci/python:3.7.3-node
+      - image: wlonk/oddbird
+    environment:
+      VIRTUAL_ENV: ./venv
```
Comment (jgerigmeyer) [inline .circleci/config.yml]: @wlonk What do you think about setting this in CircleCI instead of repeating it here?

Comment (wlonk) [reply .circleci/config.yml]: Unless the value is secret, I much prefer setting it in the config, so it can be in source control and tracked.

**Pull request merged**